package.json

```diff
@@ -35,125 +35,126 @@
     "prepare-vue3": "./scripts/prepare-vue3.js"
   },
   "devDependencies": {
-    "@algolia/cache-common": "4.14.3",
-    "@algolia/cache-in-memory": "4.14.3",
-    "@algolia/client-search": "4.14.3",
-    "@algolia/logger-common": "4.14.3",
-    "@algolia/requester-node-http": "4.14.3",
-    "@algolia/transporter": "4.14.3",
-    "@babel/cli": "7.15.4",
-    "@babel/core": "7.15.5",
-    "@babel/plugin-proposal-class-properties": "7.14.5",
-    "@babel/plugin-syntax-dynamic-import": "7.8.3",
-    "@babel/plugin-transform-react-constant-elements": "7.9.0",
-    "@babel/plugin-transform-runtime": "7.15.0",
-    "@babel/preset-env": "7.15.6",
-    "@babel/preset-react": "7.14.5",
-    "@babel/preset-typescript": "7.15.0",
-    "@googlemaps/jest-mocks": "2.7.5",
+    "@algolia/cache-common": "^4.14.3",
+    "@algolia/cache-in-memory": "^4.14.3",
+    "@algolia/client-search": "^4.14.3",
+    "@algolia/logger-common": "^4.14.3",
+    "@algolia/requester-node-http": "^4.14.3",
+    "@algolia/transporter": "^4.14.3",
+    "@babel/cli": "^7.15.4",
+    "@babel/core": "^7.15.5",
+    "@babel/plugin-proposal-class-properties": "^7.14.5",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/plugin-transform-react-constant-elements": "^7.9.0",
+    "@babel/plugin-transform-runtime": "^7.15.0",
+    "@babel/preset-env": "^7.15.6",
+    "@babel/preset-react": "^7.14.5",
+    "@babel/preset-typescript": "^7.15.0",
+    "@googlemaps/jest-mocks": "^2.7.5",
     "@microsoft/api-extractor": "7.18.0",
```

Comment:
this one is not with caret as later versions seem to break the build, we can update it later